mod/wet4/views/default/page/elements/message_dd_list.php

```diff
@@ -20,7 +20,7 @@
 		echo $no_results();
 		return;
 	}
-	echo "<p class='elgg-no-results message-dd-no-results'>$no_results</p>";
+	echo "<p class='elgg-no-results message-dd-no-results' aria-live='assertive'>$no_results</p>";
```

Comment:
`aria-live` is to indicate that there is dynamic content that will change. This doesn't make sense for a paragraph with static results